R/IDateTime.R

```diff
@@ -240,6 +240,30 @@ rep.ITime = function (x, ...)
   class(y) = "ITime"   # unlass and rep could feasibly not copy, hence use class<- not setattr()
   y
 }
+                           
+round.ITime <- function(x, units = c("hours", "minutes")) 
+{
+  (setattr(switch(match.arg(digits),
+                  hours = as.integer(round(unclass(x)/3600)*3600),
+                  minutes = as.integer(round(unclass(x)/60)*60)), 
+           "class", "ITime"))
+} 
+
+trunc.ITime <- function(x, units = c("hours", "minutes")) 
+{
+  (setattr(switch(match.arg(units),
+                  hours = as.integer(trunc(unclass(x)/3600)*3600),
```

Comment:
my guess is `x %/% 3600L` will be faster than `trunc(unclass(x)/3600)`. `ITime` value is restricted to `0:86399` and the two are the same in that range.

Reply:
I just did a benchmark, and trunc(unclass(x)/3600) is slightly faster.

![image](https://user-images.githubusercontent.com/37183160/73526763-2dfee100-4412-11ea-931e-451e4523f2d4.png)

Reply:
I think it depends on your input 🙃 

```
> x = structure(sample(86399L, 1e7, TRUE), class = 'ITime')
> microbenchmark::microbenchmark(
+   times = 1000L,
+   trunc(unclass(x)/3600),
+   x %/% 3600L
+ )
Unit: milliseconds
                   expr      min       lq     mean   median       uq       max neval cld
 trunc(unclass(x)/3600) 37.54874 39.88270 48.46855 50.45054 53.37909 246.11452  1000   b
              x%/%3600L 40.50842 42.40509 47.17708 43.23612 54.86358  71.76187  1000  a 
```

Reply:
Yeah okay. So you suggest that I exchange `trunc()` with `%/%`.

My reasoning was merely to be consistent in the use of `base` functions :-)

Reply:
I will change from `trunc` to `%/%`